docs/upgrading.md

```diff
@@ -5,6 +5,10 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.3.1` to `v0.3.2`
+
+No backwards incompatible changes
```

Comment:
100% sure?

Reply:
Yes, we have not updated any things in the framework except for the dev mode which was a fix